dll/win32/newdev/lang/zh-CN.rc

```diff
@@ -85,13 +86,13 @@ BEGIN
     LTEXT "安装完成", IDC_FINISHTITLE, 120, 8, 195, 12
     LTEXT "向导已完成安装的驱动程序:", IDC_STATIC, 120, 32, 195, 16
     LTEXT "单击完成关闭向导。", IDC_STATIC, 120, 174, 195, 12
-    LTEXT "未知的设备", IDC_DEVICE, 148, 53, 147, 12
+    LTEXT "未知设备", IDC_DEVICE, 148, 53, 147, 12
 END
 
 STRINGTABLE
 BEGIN
-    IDS_INSTALLWIZARDTITLE "新的硬件向导"
-    IDS_UPDATEWIZARDTITLE "更新驱动程序向导"
+    IDS_INSTALLWIZARDTITLE "找到新的硬件向导""
```

Comment:
You have a double `"` at the end of this text, which makes the build fail.

Reply:
Corrected.